Core.Collectors/Config/ConfigManager.cs

```diff
@@ -156,6 +161,32 @@ protected JToken GetAuthenticationToken(string collectorType)
             return authenticationToken;
         }
 
+        public JToken GetApiDomainToken()
+        {
+            ValidateSettingsExist();
+
+            if (!this.apiDomainTokenMap.TryGetValue(DefaultKey, out JToken apiDomainToken))
+            {
+                apiDomainToken = this.apiDomainTokenMap[DefaultKey];
+            }
+            return apiDomainToken;
+        }
+
+        public string GetApiDomain()
+        {
+            string apiDomain = string.Empty;
+            JToken apiDomainToken = this.GetApiDomainToken();
+            try
+            {
+                apiDomain = apiDomainToken.ToString();
```

Comment:
```apiDomainToken.Value<string>()```

For consistency with the rest of the code.

Reply:
Good catch, I was searching for the proper way to convert to a primitive type based on the rest of the code and couldn't find an example.